test/isImplementedCache.test.ts

```diff
@@ -14,22 +14,28 @@ describe('ImplementedCache', () => {
         isImplementedCache.add(event, true);
 
         expect(isImplementedCache.get(event)).toBeDefined();
-        expect(isImplementedCache.get(event)?.isImplemented).toBeTrue();
+        expect(isImplementedCache.get(event)?.isImplemented).toBe(true);
```

Comment:
Why was this changed ?

Reply:
toBeTrue was flagging as an error on my Editor that is why I changed it. i have reverted my changes